modules/acquia_cms_tour/src/Services/StarterKitService.php

```diff
@@ -91,6 +122,16 @@ public function enableModules(string $starter_kit, string $demo_question = NULL,
       ->save();
   }
 
+  /**
+   * Handler for enabling modules.
+   *
+   * @param string $module
+   *   Variable holding the module name.
+   */
+  public static function enableModule(string $module) {
+    \Drupal::service('module_installer')->install([$module]);
```

Comment:
@rajeshreeputra  As this file is already a service then please Use D.I (use Drupal\Core\Extension\ModuleInstallerInterface;)

Reply:
in static function DI isn't allowed